pkg/webview/view.proto

```diff
@@ -108,10 +108,35 @@ message View {
 
 message GetViewRequest {}
 
+message SnapshotHighlight {
+  string beginning_log_id = 1;
```

Comment:
i think this needs to be beginning_logID to preserve the old casing? surprised the test didn't catch this

Reply:
Ah well the test doesn't have a highlight so I don't think it would. I'll add a highlight to the test and fix this.